Makefile

```diff
@@ -75,6 +75,7 @@ report: init info tests
 	./tools/sv-report
 	cp ./conf/report/report.* ./out/.
 	cp -rfT ./tests ./out/tests/
+	find ./out/tests -name *.sv -exec ./tools/formatter {} +
```

Comment:
Do we really need to copy the test first? Do we use the non-html test in any way in the report?

Reply:
BTW. you shouldn't use `./tests` directly, use the `TESTS_DIR` variable.

Reply:
There is no real need for the raw `.sv` files, I will modify this to take tests from `TESTS_DIR` and only save the `.html` versions inside the `./out` directory

Reply:
Current version works without copying the files first and uses the proper variables